src/components/pages/DataViz/RenderMap.js

```diff
@@ -141,70 +145,104 @@ const RenderMap = () => {
 
     setDetailsData(bridge);
   };
+  let screenHeight = '90vh';
```

Comment:
place these styling inside a scss file

Reply:
I had to do this on purpose. Since the className doesn't work on an API tag, I had to make it as an object.